django_react_templatetags/context_processors.py

```diff
@@ -4,6 +4,7 @@
 
 def react_context_processor(request):
     """Expose a global list of react components to be processed"""
+    print("react_context_processor is no longer required.")
```

Comment:
There might be a better way to handle this :-)

Reply:
This should probably be the way to go: https://docs.python.org/3/library/exceptions.html#DeprecationWarning